.github/workflows/docker.yml

```
@@ -62,19 +62,77 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-        # This should work without any of the project sources except compose.yaml
         with:
-          sparse-checkout: |
-            compose.yaml
-          # Disable cone mode so we only grab a specific file
-          # https://github.com/actions/checkout?tab=readme-ov-file#fetch-only-a-single-file
-          # https://git-scm.com/docs/git-sparse-checkout
-          sparse-checkout-cone-mode: false
+          path: checkout
+      - uses: dtolnay/rust-toolchain@stable
+      - run: cargo build --package divviup-cli --profile release
+        working-directory: checkout
+      - run: |
+          mkdir compose
+          cp checkout/target/release/divviup compose/
+          cp checkout/compose.yaml compose/
+      # ubuntu-latest is 22.04, which does not package jq newer than 1.6.1, so grab a binary from
+      # GitHub releases
+      - name: install jq 1.7
+        run: |
+          curl -Lso jq https://github.com/jqlang/jq/releases/download/jq-1.7.1/jq-linux-amd64
+          chmod +x jq
+          ./jq --version
       - name: Compose
         id: compose
         run: docker compose up --wait --wait-timeout 120
+        working-directory: compose
+      - name: Demo script
+        id: demo-script
+        working-directory: compose
+        run: |
+          export DIVVIUP_API_URL=http://localhost:8080
+          export DIVVIUP_TOKEN=""
+
+          export DIVVIUP_ACCOUNT_ID=`./divviup account list | ../jq -r '.[0].id'`
+          printf 'account ID %s\n' $DIVVIUP_ACCOUNT_ID
+
+          AGGREGATOR_LIST=`./divviup aggregator list`
+          printf 'aggregator list %s\n' $AGGREGATOR_LIST
+
+          LEADER_ID=`echo $AGGREGATOR_LIST | ../jq -r 'map_values(select(.name == "leader")).[0].id'`
```

FYI, it looks like we could lower `map_values` as follows, to avoid needing to install a newer version.
```suggestion
          LEADER_ID=`echo $AGGREGATOR_LIST | ../jq -r '.[] |= select(.name == "leader") |.[0].id'`
```